### ISSUE TITLE
State of the project

### ISSUE THREAD

**Issue by ctron** [KenDJohnson/cpe-rs#13]
I would like to understand the state of the project. And I don't want to push in any direction.

I absolutely understand of priories change, or if my PRs aren't good enough :)

On the other side, there are a few changes I (our team) needs. I would prefer it if those changes land upstream, in this repository so that everyone can consume them. But if that doesn't work out, I am ok with forking the repository as well.

Still, I wanted to try and reach out so see if there's a way to keep this repository active.

**Comment by jobs62** [comment]
I'm in the same situation where i (my unipersonal team lol) am looking for a crate that would allow me to manipulate cpe. cpe-rs have 80-ish % of what i'm looking for. I can develop the last few feature that i need and it would be better upstream that on my local fork-ish.

**Comment by KenDJohnson** [comment]
Sorry, I'm not on my public github all that often, I'll try to keep a better eye on it.

@jobs62 I'm happy to get PRs for any changes you want to upstream.

> Still, I wanted to try and reach out so see if there's a way to keep this repository active.

@ctron you had previously mentioned the idea of creating a group/org for some creates like this, is that something you're still considering?

**Comment by ctron** [comment]
> @ctron you had previously mentioned the idea of creating a group/org for some creates like this, is that something you're still considering?

Yes. However, there's interest, but not too much progress. Everyone's busy and "distracted". But it's still on the table.

**Comment by ctron** [comment]
@KenDJohnson It's back on the table, with a bit of time allocated for this: https://github.com/scm-rs

If you want to bring `cpe-rs`, this would mean:

* we transfer the repository over to that organization
* you become a `maintainer` on the organization and can commit & publish on all crates of this organization (if that is something that you wish to do)
* releases are performed via the shared github workflow, which means adding `scm-rs/maintainers` to crates.io.

Where "organization" means "GitHub organization". If there's interest and we find an open source foundation accepting this bag of crates, then we could move that all over "there".

**Comment by KenDJohnson** [comment]
@ctron That all sound good and reasonable to me. I'm out on parental leave at the moment, but I should be able to get to this next month.
